contracts/AirbroFactory.sol

```diff
@@ -11,7 +11,7 @@ contract AirbroFactory {
     // index of deployed airdrop contracts
     address[] public airdrops;
     uint256 public totalAirdropsCount = 0;
-    address public admin = 0xF4b5bFB92dD4E6D529476bCab28A65bb6B32EFb3;
```

Comment:
Is there any particular reason you moved this initialization from this field to the constructor even though it's not passed as a parameter it seems to me it would make sense to keep it like this?

Reply:
Is there a preferred method of doing this? Regarding price, gas etc?

Reply:
I think assigning in the constructor makes sense only if you are passing the variable as a parameter to the constructor otherwise it's simpler to have it in one line declaration. This is nitpicking really but I just prefer less lines of code when possible 😄